docs/source/cloud-setup/policy.rst

```diff
@@ -197,26 +230,51 @@ Enforce Autostop for all Tasks
         @classmethod
         def validate_and_mutate(
                 cls, user_request: sky.UserRequest) -> sky.MutatedUserRequest:
-            """Enforces autostop for all tasks."""
+            """Enforces autostop for all tasks.
+            
+            Note that with this policy enforced, users can still change the autostop
+            setting for an existing cluster by using `sky autostop`.
+            """
             request_options = user_request.request_options
+
             # Request options is None when a task is executed with `jobs launch` or
             # `sky serve up`.
             if request_options is None:
                 return sky.MutatedUserRequest(
                     task=user_request.task,
                     skypilot_config=user_request.skypilot_config)
+
+            # Get the cluster record to operate on.
+            cluster_record = sky.status(request_options.cluster_name, refresh=True)
+
+            # Check if the user request should specify autostop settings.
+            need_autostop = False
+            if not cluster_record:
```

Comment:
is None

Reply:
Also this code seems different from the source.

Reply:
minor: Does sphinx have a quick way to link to source & expand?

Reply:
Good point! Changed to `literalinclude` to make it directly from source code for single source of truth.